addresspoints/src/main/scala/grasshopper/addresspoints/model/Status.scala

```diff
@@ -0,0 +1,3 @@
+package grasshopper.addresspoints.model
+
+case class Status(status: String, service: String, time: String, host: String)
```

Comment:
Also, we currently have separate classes for the the addresspoint and census `Status` models.  Should we merge them so we can guarantee all `/status` will always return the same format.

I guess the same goes for the `/status` resource itself.  Could we just "import" that into each API we build.  I don't know enough about Akka HTTP to know how easy that would be.

...and to now argue with myself, I guess the drawback is the services are would then share code, and I've heard sharing models between microservices _can_ be seen as an antipattern since you've now coupled them with a shared dependency.  Curious to get your take on that.